src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -68,11 +68,11 @@ function gotStream(stream) {
     var filteredStream = webAudio.applyFilter(stream);
 
     var servers = null;
-
-    pc1 = new webkitRTCPeerConnection(servers); // jscs:ignore requireCapitalizedConstructors
+    /*eslint new-cap: 0*/
```

Comment:
Could you try `//eslint-disable-line no-use-before-define` from http://stackoverflow.com/questions/27732209/turning-off-eslint-rule-for-a-specific-line/29592334#29592334 ?